Tests/FluentAssertions.Specs/Primitives/BooleanAssertionSpecs.cs

```diff
@@ -15,6 +15,7 @@ public void Should_succeed_when_asserting_boolean_value_true_is_true()
             bool boolean = true;
 
             // Act / Assert
+            // ReSharper disable once ConditionIsAlwaysTrueOrFalse
```

Comment:
I like how you did this in `NullableBooleanAssertionSpecs.cs`

Reply:
Fixed in [899d468](https://github.com/fluentassertions/fluentassertions/pull/2470/commits/899d46880132f777a1a979dd4fa23a357b5a30ae).